ts/sagas/contentLoaders.ts

```diff
@@ -19,11 +19,14 @@ const contentClient = ContentClient();
  */
 function getServiceMetadata(
   serviceId: ServiceId
-): Promise<BasicResponseType<ServiceMetadata> | undefined> {
+): Promise<Either<Error, BasicResponseType<ServiceMetadata>>> {
```

Comment:
also make this `t.Validation< ServiceMetadata > | undefined` (see https://github.com/teamdigitale/io-ts-commons/blob/master/src/requests.ts#L105)